Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,30 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            // To do: call your implementation.
+            int age = Lesson4.promptInt(message);
```

Comment:
small suggestion: You can probably return - 
`return Lesson4.promptInt(message)`
This will save you 1 line of code in each of the methods.